README_GCP.md

```diff
@@ -41,26 +41,18 @@ gcloud iam service-accounts add-iam-policy-binding opentelemetry-demo@${GCLOUD_P
     --member "serviceAccount:${GCLOUD_PROJECT}.svc.id.goog[otel-demo/opentelemetry-demo-otelcol]"
 ```
 
-Update [`gcp-config-valus.yml`](gcp-config-values.yml) to annotate the
-Kubernetes service account with your project:
+### Deploying the Helmfile
+
+Without Workload Identity, run
 
 ```console
-sed -i "s/%GCLOUD_PROJECT%/${GCLOUD_PROJECT}/g" gcp-config-values.yml
+helmfile --interactive apply -f gcp/helmfile.yaml
```

Comment:
Why add the interactive flag here?

Reply:
So the user is prompted to accept the diff before it is executed. I figured since these are instructions for a human, it's reasonable to prompt for confirmation.